src/vcpkg/commands.z-preregister-telemetry.cpp

```diff
@@ -62,7 +62,9 @@ namespace vcpkg
         }
         else
         {
-            msg::println_warning(msgVcpkgSendMetricsButDisabled);
+            msg::write_unlocalized_text_to_stderr(
+                Color::warning,
+                LocalizedString::from_raw(WarningPrefix).append(msgVcpkgSendMetricsButDisabled).append_raw('\n'));
```

Comment:
Maybe 
stderr_sink.println_warnin
```suggestion
            stderr_sink.println_warning(msgVcpkgSendMetricsButDisabled);
```

Reply:
I don't want to introduce use of the sinks where the polymorphism is not needed right now but I could see an argument in the future that all console output should be polymorphic like this. (But that seems way outside the scope of this PR)